internal/node-patches/test/subprocess/worker-threads-script.ts

```diff
@@ -0,0 +1,49 @@
+const {
+  Worker,
+  isMainThread,
+  parentPort,
+  workerData,
+} = require('worker_threads');
```

Comment:
any particular reason this doesn't use `import`? either change to import or comment why it's different

Reply:
this was copy pasted from the nodejs docs example so happy to clean it.